x-pack/plugins/observability_solution/investigate_app/public/hooks/use_fetch_investigation_items.ts

```diff
@@ -45,20 +46,22 @@ export function useFetchInvestigationItems({ investigationId, initialItems }: Pa
 
   const { isInitialLoading, isLoading, isError, isSuccess, isRefetching, data, refetch } = useQuery(
     {
-      queryKey: investigationKeys.fetchItems({ investigationId }),
+      queryKey: investigationKeys.detailItems(investigationId),
       queryFn: async ({ signal }) => {
         return await http.get<GetInvestigationItemsResponse>(
           `/api/observability/investigations/${investigationId}/items`,
           { version: '2023-10-31', signal }
         );
       },
-      initialData: initialItems,
+      // initialData: initialItems,
```

Comment:
I think you meant to remove the commented line?

Reply:
yes I'm not 100% sure it's useful since we are in the same context, React Query already has it in the cache anyway